packages/parameterized_test/test/util/cast_helper_test.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:parameterized_test/src/errors/parameter_type_error.dart';
+import 'package:parameterized_test/src/util/cast_helpers.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
+  group('Single value casting successful', () {
```

Comment:
Nice that you test casting!

Can you just plainly cast everything? Even my custom object? I think it should right? Either
- add a test that shows you can cast a 'custom' class object
- address the limits of casting in readme (only int/string/... supported)

Reply:
CustomObject is possible. Not sure if it should be mentioned in readme